src/server/main_service.cc

```diff
@@ -1358,6 +1358,10 @@ bool Service::InvokeCmd(const CommandId* cid, CmdArgList tail_args, SinkReplyBui
   // TODO: we should probably discard more commands here,
   // not just the blocking ones
   const auto* conn = cntx->conn();
+  if (cntx->conn_state.squashing_info) {
```

Comment:
what is the reason we dont save the connection pointer in cntx when running squashed commands?

Reply:
i dunno. maybe @dranikpg  remembers.